### DISCUSSION TITLE
How do you get the position index of the table on the page

### DISCUSSION THREAD

**Discussion by Godlikemandyy** [jsvine/pdfplumber#849] [Q&A] [answered]
I could use `page.extract_table()` obtain texts, but I want to get the table coordinates in page. 
So I can insert the extracted table into the whole page of text. How can I get the coordinates of the table.

**Comment by jsvine** [comment, marked as answer]
Hi @Godlikemandyy, you can use `tables = page.find_tables()` to get the table objects, and then `table.bbox` to get the table's coordinates.

**Reply by Godlikemandyy** [reply]
I could get the coordinates of the table in the way you said. I also want to ask `(113.04899999999999, 476.728, 442.192, 652.168)`, 
What are the variables represented by these four values？

**Reply by jsvine** [reply]
These correspond to the `x0`, `top`, `x1`, `bottom` attributes of the table.

**Reply by Godlikemandyy** [reply]
Roger that, thanks！
